proposals/lip-0035.md

```diff
@@ -67,9 +66,9 @@ The state of an account on the Lisk mainnet after applying the block at height `
 
 In order to ensure the uniqueness of the account state, we perform the following additional checks:
 
-* Ensure that an account only has an associated public key if and only if there is at least one outgoing transaction from that account.
+* Ensure that an account has a 20-byte address if there is at least one outgoing transaction from that account and an 8-byte address otherwise.
```

Comment:
Here and in several other places, should it be, e.g., 20-byte address or 20-bytes address?

Also, just to be sure, there is no way someone received a transaction in a 20-byte address because the new address system comes after regenesis, right?

Reply:
I think 20-byte value is correct, similar to 5-day week or 32-bit value.

Yep, there are no transactions to a 20-byte address because the new address system is introduced after regenesis.